packages/realm_dart/lib/src/subscription.dart

```diff
@@ -112,29 +115,25 @@ enum SubscriptionSetState {
 /// Realm is an expensive operation server-side, even if there's very little data that needs
 /// downloading.
 /// {@category Sync}
-abstract class SubscriptionSet with IterableMixin<Subscription> implements Finalizable {
+sealed class SubscriptionSet with Iterable<Subscription> implements Finalizable {
   final Realm _realm;
-  SubscriptionSetHandle _handle;
+  SubscriptionSetHandle __handle;
```

Comment:
It's a small thing, but maybe the difference between __handle and _handle is a little difficult to see.

Reply:
I know, but it is only used in the 4 lines following, otherwise we always use `_handle` or `SubscriptionInternal.handle` extension method.